react/ProductList.tsx

```diff
@@ -56,7 +56,7 @@ const ProductList: FC<Props> = ({ renderAsChildren, classes }) => {
         })
       }
 
-      updateQuantity({ uniqueId, quantity }, options)
+      updateQuantity({ index: item.index, quantity }, options)
```

Comment:
Is it safe to say that `index` always exists?

Reply:
yep, because the uniqueId is transformed in an index into the API